spec/03-plugins/17-jwt/01-jwt_parser_spec.lua

```diff
@@ -76,7 +76,29 @@ describe("Plugin: jwt (parser)", function()
       local token = jwt_parser.encode({sub = "1234"}, "secret", nil, {alg = "RS256"})
       assert(jwt_parser:new(token))
     end)
+
+    it("accepts a valid RS512 encoding request", function()
+      local token = jwt_parser.encode({sub = "1234"}, "secret", nil, {
+        typ = "JWT",
+        alg = "RS512"
```

Comment:
minor style nitpick, its preferred to leave a trailing comma after the last item in a table to facilitate cleaner future commit histories.